appveyor.yml

```diff
@@ -1,5 +1,5 @@
 environment:
-  APPVEYOR_RDP_PASSWORD: juliadebugging42
+  APPVEYOR_RDP_PASSWORD: juliadebugging42PLEASE
```

Comment:
![giphy](https://user-images.githubusercontent.com/1282691/94312207-c38c4980-ff7c-11ea-8196-3fcde86a99d8.gif)

Reply:
It's only weird if it doesn't work